app/grants/views.py

```diff
@@ -635,6 +635,12 @@ def grant_fund(request, grant_id, grant_slug):
             subscription.active = True
             subscription.subscription_hash = request.POST.get('subscription_hash', '')
             subscription.contributor_signature = request.POST.get('signature', '')
+            if 'split_tx_id' in request.POST:
+                subscription.split_tx_id = request.POST.get('split_tx_id', '')
+                subscription.save_split_tx_to_contribution()
+            if 'split_tx_confirmed' in request.POST:
```

Comment:
@danlipert i made a  proposed fix for this  https://github.com/gitcoinco/web/pull/5769